bradybridges/jetcake-react-app
# Config

#### What's this PR do?
Adds dependencies for router and jest/enzyme. Configures adapter in setupTests for snapshots. Add PR template to the docs directory.
#### How should this be manually tested?
Run `npm start`, you should see the barebones react application. Run `npm test` you should see that jest runs successfully 
#### What are the relevant tickets?
#1 
#### Screenshots (if appropriate)
<img width="335" alt="Screen Shot 2020-01-27 at 1 49 24 PM" src="https://user-images.githubusercontent.com/25031031/73212632-dc9ddb80-410b-11ea-9ae7-b56f5e46776f.png">
